site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -0,0 +1,632 @@
+
+Here's an overview of the directories:
+
+**src/** 
+
+- The **src/** directory contains the source code for the app. This includes all stored procedures, user-defined functions (UDFs), and the front-end for the app (in this Quickstart, the front-end files are within the **ui/** folder). It also includes two other required (and very important) files: **manifest.yml** and **setup.sql**. We'll take a deeper look at this directory in the next step.
+
+**prepare/** 
+
+- This directory is specific to this Quickstart. It contains files to prepare your Snowflake account by creating of roles, granting privileges, and loading data. We'll execute the scripts in this folder before building the native app.
+
+**consumer/** 
+
+- This directory is specific to this Quickstart. It contains a SQL script to install the native app in the account and grant appropriate privileges. This will allow you to run the native app from the perspective of the consumer.
+
+**provider/** 
+
+- This directory is specific to this Quickstart. It contains a SQL script to create the application package and grant privileges on provider data.
+
+<!-- ------------------------ -->
+## The **src/** directory
+Duration: 3
+
+Let's take a deeper look at the **/src** directory for this Quickstart.
+
+```plaintext
+src/
+├─ README.md
+├─ manifest.yml
+├─ setup.sql
+├─ ui/
+│  ├─ chairlift_data.py
+│  ├─ environment.yml
+│  ├─ first_time_setup.py
+│  ├─ references.py
+│  ├─ ui_common.py
+│  ├─ util.py
+│  ├─ v_configuration.py
+│  ├─ v_dashboard.py
+│  ├─ v_sensor_data.py
+```
+
+This directory contains the source code for the native app. This Quickstart uses **src/** as the name of the folder, mostly for the purposes of convention. In practice, this folder may take on any name you'd like. Here's an overview of what this folder contains:
+
+**manifest.yml** 
+
+- A manifest file is a requirement when creating a native app. This file defines the configuration and setup properties required by the application. It contains metadata about the app (version, etc.), artifacts required by the app, and log configuration settings. It also defines the privileges that the consumer must grant when the application is installed and run in their account. Finally, it also contains references defined by the provider. Typically these references refer to tables and the corresponding privileges needed by the app to run against consumer data. For more information, see [Creating the Manifest File](https://docs.snowflake.com/en/developer-guide/native-apps/creating-manifest).
```

Comment:
I'd change the term "setup properties", I think you can bucket everything as "runtime configuration".